mgmtsystem/__manifest__.py

```diff
@@ -39,5 +23,5 @@
         'views/res_config.xml'
     ],
     "demo": [],
-    'installable': False,
+    'installable': True,
 }
```

Comment:
Hi, In the changelog it's written "the module does no depends anymore on document_page module", however in the manifest the dependency on document_page is present. Which one is correct?